Day2/Exercise1/Solution.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "fmt"
+
+func Frequency(s string) map[rune]int {
+	m := map[rune]int{}
+	for _, r := range s {
+		m[r]++
+	}
+	return m
+}
+func countFreq(stream []string) map[rune]int {
+
+	sum := map[rune]int{}
+	count := len(stream)
+	results := make(chan map[rune]int, count)
+
+	for _, s := range stream {
+		go func(s string) {
+			results <- Frequency(s)
+		}(s)
+	}
+
+	for i := 0; i < count; i++ {
+		for r, freq := range <-results {
+			sum[r] += freq
+		}
+	}
+
+	return sum
+}
+func main() {
+	stream := make([]string, 5)
+	stream[0]="quick"
```

Comment:
Instead of inserting the elements one by one , you can do it in one go , that will make the code smaller as well as save your time